src/System Application/App/ClientExtensions/app.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
We now have https://github.com/microsoft/BCApps/tree/main/src/System%20Application/App/ControlAddIns. Please put these there instead.

Reply:
When they are moved, change the namespace as well.
Check with Andreas on what namespace(s) they should fall into.